client/src/models/Address.ts

```diff
@@ -0,0 +1,14 @@
+export interface Address {
+    city: string;
+    neighborhood?: string;
+    street?: string;
+    houseNumber?: string;
```

Comment:
Isn't these parameters should be required?

Reply:
יש קיבוצים שאין כתובות ממש של מספרים ורחובות, וזה רק עיר, זה יותר עניין אפיוני בגדול